sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -188,19 +188,25 @@ def test_pardo_windowed_side_inputs(self):
               (9, list(range(7, 10)))]),
           label='windowed')
 
-  def test_flattened_side_input(self):
+  def test_flattened_side_input(self, with_transcoding=True):
     with self.create_pipeline() as p:
       main = p | 'main' >> beam.Create([None])
       side1 = p | 'side1' >> beam.Create([('a', 1)])
       side2 = p | 'side2' >> beam.Create([('b', 2)])
-      side3 = p | 'side3' >> beam.Create([('b', 3)])
+      if with_transcoding:
+        # Also test non-matching coder types (transcoding required)
+        side3 = p | 'side3' >> beam.Create([('another_type')])
```

Comment:
I would assign this to a variable, say, `third_element`, and then here and below refer to that rather than have the conditionals.

Reply:
Thanks. Looks much better.